charts/nri-bundle/values.yaml

```diff
@@ -3,9 +3,14 @@
 # Declare variables to be passed into your templates.
 
 # enables newrelic-infrastructure
+# Legacy nri-kubernetes chart, disabled by default
```

Comment:
This comment is lying, and I don't remember why I set the old one to `true` and the new to `false`. Could we invert this logic?

Reply:
Do we want to have the new one enabled by default even during the beta?

Reply:
I would change the comment but not "the logic" until we are really sure that we want the v3 as the default one.